packages/bridge-ui/src/App.svelte

```diff
@@ -251,7 +251,7 @@
   };
 
   const routes = {
-    "/": wrap({
+    "/:tab?": wrap({
```

Comment:
I have a question about the `wrap()` function. Do we really need to wrap the router if we're not using any of the features? I guess it doesn't make any harm, just curiosity.

Reply:
We can still use it as it enables code splitting. Sets a good tone for future routes as well reduces size per request for the current bridge UI.

Reply:
Got it. Thanks